### PR TITLE
Including batch_norm as the normalizer function by default, as mentioned in function description

### PR DESCRIPTION
**Pull request by kwotsin**
Changed the description error that includes `batch_norm_params`, which no longer exists, to `normalizer_fn`. `normalizer_fn = None` is changed to `normalizer_fn = batch_norm` by default, following what the original description says the function will do.

### CODE REVIEW

File: tensorflow/contrib/layers/python/layers/layers.py [hunk 1]
```diff
@@ -1831,7 +1831,7 @@ def separable_convolution2d(
     padding='SAME',
     rate=1,
     activation_fn=nn.relu,
-    normalizer_fn=None,
+    normalizer_fn=batch_norm,
```
Comment (fchollet) [inline tensorflow/contrib/layers/python/layers/layers.py]: This is meant to be None. Every other layer (in particular conv layers) defaults to None.

Such a change would also break many models.

**Review by fchollet — CHANGES_REQUESTED** [review]

**Review by fchollet — APPROVED** [review]

**Comment by drpngx** [comment]
Jenkins, test this please.
